packages/discovery-react-components/src/components/ResultsPagination/ResultsPagination.tsx

```diff
@@ -34,6 +44,10 @@ export interface ResultsPaginationProps {
    * custom handler invoked when any input element changes in the ResultsPagination component
    */
   onChange?: (e: ResultsPaginationEvent) => void;
+  /**
+   * ref passed in by parent that can be used to reset Carbon's pagination component
```

Comment:
Don't mention the carbon component; that's an implementation detail that shouldn't be exposed.